nixos/modules/services/x11/xserver.nix

```diff
@@ -694,7 +647,7 @@ in
       [ xorg.xorgserver.out
         xorg.xrandr
         xorg.xrdb
-        xorg.setxkbmap
+        xorg.setxkbmap # TODO: does this work under wayland?
```

Comment:
TODO: resolve this by either moving the setting or dropping the comment

Reply:
When i run that blindly against a wayland session it prints 

```log
[nix-shell:~]$ setxkbmap 
WARNING: Running setxkbmap against an Xwayland server
```

so i kind of assume that its mean as xserver only, possibly we could add it when enabling xwayland (but dont know it that make sense).

Reply:
Maybe we could depend it on ``programs.xwayland.enable`` but since that is a rather small convince issue, I would leave that for now open. Thanks!